### PR TITLE
Includes DT_STATSD_INGEST_URL in the endpoint.properties file injected to pods

### PR DESCRIPTION
**Pull request by toszr**
If `statsd-ingest` capability is enabled for ActiveGate, `endpoint.properties` file is enhanced with 
```
DT_STATSD_INGEST_URL=dynakube-activegate.dynatrace:18125
```

It's used to discover StatsD service endpoint in the cluster (similar to `metrics-ingest` except for StatsD there is no token).

### CODE REVIEW

File: src/controllers/activegate/capability/statsd-ingest/consts.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package statsdingest
+
+import "github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/internal/consts"
+
+const (
```
Comment (chrismuellner) [inline src/controllers/activegate/capability/statsd-ingest/consts.go:5]: for the rest of the codebase we have the constants in a `config.go` per package

Comment (toszr) [reply src/controllers/activegate/capability/statsd-ingest/consts.go]: Makes sense. I rearranged StatsD- and other ActiveGate-related constants as suggested.

**Review by 0sewa0 — COMMENTED** [review]
looks good, there is a conflict in the tests

**Comment by toszr** [comment]
> looks good, there is a conflict in the tests

Yikes! Had to move the constants to `activegate/capability/config.go` to avoid package name clash. 

Might make sense to rethink ActiveGate file and package structure, similar to what @0sewa0 did with other controllers (why is there a `controllers/activegate` and `controllers/dynakube/activegate`?). Something in the plans, @mjgrzybek @aorcholski?

**Review by 0sewa0 — APPROVED** [review]

**Pull request merged**